index.js

```diff
@@ -87,14 +89,46 @@ module.exports = function loader(content) {
       }
     });
 
+    if (outputPlaceholder) {
+      q.defer(function generatePlaceholder(queueCallback) {
+        img
+            .clone()
+            .resize(placeholderSize, jimp.AUTO)
+            .quality(quality)
+            .background(background)
+            .getBuffer(mime, function resizeCallback(queueErr, buf) {
+              if (err) {
+                return queueCallback(queueErr);
+              }
+
+              const placeholder = buf.toString('base64');
+              return queueCallback(null, JSON.stringify('data:' + (mime ? mime + ';' : '') + 'base64,' + placeholder));
+            });
+      });
+    }
+
     return q.awaitAll((queueErr, files) => {
+      'use strict'; // eslint-disable-line
```

Comment:
Thought so too, but it was failing with v4 and v5, only passed with v6.
Let me look into this a bit more...